core/core_bind.cpp

```diff
@@ -557,6 +557,33 @@ String OS::get_unique_id() const {
 	return ::OS::get_singleton()->get_unique_id();
 }
 
+void get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) {
+	String pf = p_function;
+	if (p_idx == 0 && pf == "has_feature") {
+		HashSet<String> presets;
+
+		presets.insert("bptc");
+		presets.insert("s3tc");
+		presets.insert("etc");
+		presets.insert("etc2");
+		presets.insert("editor");
+		presets.insert("template_debug");
+		presets.insert("template_release");
+		presets.insert("debug");
+		presets.insert("release");
+		presets.insert("template");
+		presets.insert("double");
+		presets.insert("single");
+		presets.insert("32");
+		presets.insert("64");
+		presets.insert("movie");
```

Comment:
I don't think this needs to be a hash map, the entries can be made unique as is and doesn't depend on data from anywhere else